src/libs/ReimbursementAccountUtils.js

```diff
@@ -35,7 +35,7 @@ function getDefaultStateForField(props, fieldName, defaultValue = '') {
 function getBankAccountFields(props, fieldNames) {
     return {
         ..._.pick(lodashGet(props, 'reimbursementAccount.achData'), ...fieldNames),
-        ..._.pick(lodashGet(props, 'reimbursementAccountDraftValues'), ...fieldNames),
+        ..._.pick(lodashGet(props, 'reimbursementAccountDraft'), ...fieldNames),
```

Comment:
I think we can remove this line, I added it in case `reimbursementAccountDraftValues` wouldn't exist in `props` in other scenarios.

```suggestion
```